setup/openchange_neworganization

```diff
@@ -41,8 +41,13 @@ parser.add_option("--firstorg", type="string", metavar="FIRSTORG",
 parser.add_option("--firstou", type="string", metavar="FIRSTOU",
                   default="First Administrative Group",
                   help="set OpenChange Administrative Group (otherwise 'First Administrative Group')")
+parser.add_option("--ignore-already-exists", action="store_true",
+                  help="Ignore errors when installing schemas that already exists")
+parser.add_option("--enable-all-groups", action="store_true",
```

Comment:
:+1: to this suggestion as one of the most advanced users is unable to find the option without reading carefully the code, then the option is mislead placed. Use `openchange_group --firstorg=foobar --enable-all-groups` seems logical.